minecraft_go/software/mc_go/include/general.h

```diff
@@ -10,5 +10,11 @@
 
 #define EXIT_OK 0
 
+#define CR_CHAR '\r'
+#define LF_CHAR '\n'
```

Comment:
These aren't used outside of GPS parsing, it's better to keep them inside `gps.h`